docs/integrate-logto/interact-with-management-api/README.md

```diff
@@ -0,0 +1,144 @@
+
+## How to access Logto Management API
+
+### Create an M2M app
+
+On the "Applications" page, select the M2M app type and start the creation process.
+
+<M2mRoleAssignment />
+
+In the role assignment module, you can see all M2M roles are included, and roles indicated by a Logto icon means that these roles include Logto Management API permissions.
+
+Now assign M2M roles include Logto Management API permissions for your M2M app.
+
+### Fetch an access token
+
+#### Basics about access token request
+
+<BasicsAboutAccessTokenRequest />
+
+#### Fetch access token for Logto Management API
+
+<FetchAccessTokenForLogtoManagementApi />
+
+#### Access token response
+
+A successful access response body would be like:
+
+```json
+{
+  "access_token": "eyJhbG...2g", // Use this token for accessing the Logto Management API
+  "expires_in": 3600, // Token expiration in seconds
+  "token_type": "Bearer", // Auth type for your request when using the access token
+  "scope": "all" // scope `all` for Logto Management API
+}
+```
+
+<M2mAccessTokenNote />
+
+### Access Logto Management API using access token
+
+<AccessTokenUsage />
+
+<AccessLogtoManagementApiUsingAccessToken />
+
+## Typical scenarios for using Logto Management API
+
+Our developers have implemented many additional features using Logto Management API. We believe that our API is highly scalable and can support a wide range of your needs. Here are a few examples of scenarios that are not possible with the Logto Admin Console but can be achieved through the Logto Management API.
+
+### Implement user profile on your own
+
+Logto currently does not provide a pre-built UI solution for user profiles. We recognize that user profiles are closely tied to business and product attributes. While we work on determining the best approach, we suggest using our APIs to create your own solution. For instance, you can utilize our interaction API, profile API, and verification code API to develop a custom solution that meets your needs. We’ve prepared a dedicated page [User Profile](/docs/recipes/user-profile/README.md) for tutorials and guides.
+
+### Advanced user search
+
+The Logto Admin Console supports basic search and filtering functions. For advanced search options like fuzzy search, exact match, and case sensitivity, check out our [Advanced User Search](/docs/recipes/manage-users/advanced-user-search/) tutorials and guides.
+
+### Implement organization management on your own
+
+If you’re using the [organization](/docs/recipes/organizations/) feature to build your multi-tenant app, you might need the Logto Management API for tasks like organization invitations and member management. For your SaaS product, where you have both admins and members in the tenant, the Logto Management API can help you create a custom admin portal tailored to your business needs. Check out [this](/docs/recipes/organizations/configuration/#configure-via-management-api) for more detail.
```

Comment:
These `/docs/recipes/...` links should be updated with new ones, as we are going to deprecate all new version docs later